content/en/docs/KRE/user/50_krt_v2/specs/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "KRT V2 YAML specs"
+description: >
+  YAML Definition of a KRT file.
+weight: 20
+---
+
+A KRT yaml file is a declarative file describing the content of the KRT.
+It has general description of the Version, a GRPC entrypoint for the Runtime Version and
+nodes that are connected between each other to form workflows that will be access through GRPC services defined on the entrypoint.
+
+Example below is directly taken from our krt v2 demo repo:
+
+```yaml
+version: classificator-v1
+krtVersion: v2 # new tagged krt version
+description: Demo email classificator for branching features.
+entrypoint:
+  proto: public_input.proto
+  image: konstellation/kre-entrypoint:latest
+config:
+  variables:
+workflows:
+  - name: classificator
+    entrypoint: Classificator
+    exitpoint: exitpoint # name must match node's
```

Comment:
This may be confusing as the node name is the same as the field name, I'd rename the node name to `exitpoint-node`, for example, for making the example more clear.